spec/factories/lectures.rb

```diff
@@ -2,6 +2,6 @@
   factory :lecture do
     name { "SWTII" }
     enrollment_key { "ruby" }
-    is_running { true }
```

Comment:
Does not the lecture need a status? Lecturer as well but the status is missing right?
But if no tests are failing without missing status, is maybe the test missing?
Could lead to misleading assumptions in further usage of the factory if is does not contain status information.

Or is it set by deafult? I'm not that firm with the factory so maybe just tic the checkmark an continue ;)

- [ ] check if status is needded in factory

Reply:
Status is not needed because the database is setting a default value. By leaving it out here, we test that the database default works.